app/Helpers/DatabaseHelper.php

```diff
@@ -89,35 +92,35 @@ public static function getCommentsMax($type){
         return $v->maxviews;
     }
 
-    public static function langId_from_short($short){
+    public static function langId_from_short($short) {
         $lang = \DB::table('languages')
             ->select('id')
             ->where('short', '=', $short)
             ->first();
 
-        if($lang){
+        if ($lang) {
             return $lang->id;
-        }else{
+        }else {
             return 0;
         }
 
     }
 
-    public static function developerId_from_developerName($developername){
+    public static function developerId_from_developerName($developername) {
         $dev = \DB::table('developer')
             ->select('id')
             ->where('name', '=', $developername)
             ->first();
 
-        if($dev){
+        if ($dev) {
             return $dev->id;
-        }else{
+        }else {
             return 0;
         }
 
     }
 
-    public static function developer_add_and_get_developerId($developername){
+    public static function developer_add_and_get_developerId($developername) {
```

Comment:
__[code_sniffer]__
* Method name "DatabaseHelper::developer_add_and_get_developerId" is not in camel caps format
* Opening brace should be on a new line

<!-- Comment by [SideCI](https://sideci.com) -->